src/Commands/Self/Plugin/InstallCommand.php

```diff
@@ -42,6 +43,32 @@ public function install(array $projects)
         }
     }
 
+    /**
+     * Import Terminus 2 plugins.
+     *
+     * @command self:plugin:import
+     * @aliases self:plugin:sync plugin:import plugin:sync
+     *
+     * @throws \Pantheon\Terminus\Exceptions\TerminusException
+     */
+    public function import()
+    {
+        $plugins_dir = getenv('HOME') . '/.terminus/plugins';
+        if (file_exists($plugins_dir) && is_dir($plugins_dir)) {
+            $plugin_dirs = glob($plugins_dir . '/*', GLOB_ONLYDIR);
+            if (!empty($plugin_dirs)) {
+                foreach ($plugin_dirs as $dir) {
+                    $project = $this->getProjectNameFromPath($dir);
+                    $this->install([$project]);
```

Comment:
What if the plugin has already been installed in T3? Could this be idempotent? Does it work as written, but just produce warnings, or does the process stop?

Reply:
It just reports the plugin is already installed.  Example output below:
```
$ t3 plugin:im
 [notice] terminus-plugin-project/terminus-autocomplete-plugin is already installed.
 [notice] terminus-plugin-project/terminus-code-plugin is already installed.
 [notice] pantheon-systems/terminus-composer-plugin is already installed.
 [notice] terminus-plugin-project/terminus-filer-plugin is already installed.
 [notice] terminus-plugin-project/terminus-site-status-plugin is already installed.
```
No harm, no foul?

Reply:
What happens if there is a T2 plugin that has no release on Packagist, e.g. https://github.com/pantheon-systems/terminus-cache-metrics (a private plugin some Pantheors clone directly into ~/.terminus/plugins)?

It's okay if these sorts of plugins are not installed by this PR, but ideally their presence would not cause the migration to halt.

Reply:
`$project = $this->getProjectNameFromPath($dir);` returns an error if composer.json is not found in the project root which does abort the migration.